<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quantensimulationsframework - QSFW</string>
  </property>
  <widget class="QPushButton" name="btn_compile">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Compile - Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_connect">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Connect - Mode</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="menubox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>671</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn_ident">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>ident</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_circuit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>90</width>
      <height>61</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>circuit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Elements</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hadamard">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>hadamard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_phase">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>phase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pauliX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>pauliX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pauliY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>pauliY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pauliZ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>pauliZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sphase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>sphase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tphase">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>tphase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_measure">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>measure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cnot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>cnot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_swap">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>swap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>cz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cphase">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>cphase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_toffoli">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>toffoli</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cswap">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>620</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>cswap</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Basics:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_qubit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>qubit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1-Qubit-Gates:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2-Qubit-Gates:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3-Qubit-Gates:</string>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>btn_ident</zorder>
   <zorder>btn_circuit</zorder>
   <zorder>label_2</zorder>
   <zorder>btn_hadamard</zorder>
   <zorder>btn_phase</zorder>
   <zorder>btn_pauliX</zorder>
   <zorder>btn_pauliY</zorder>
   <zorder>btn_pauliZ</zorder>
   <zorder>btn_sphase</zorder>
   <zorder>btn_tphase</zorder>
   <zorder>btn_measure</zorder>
   <zorder>btn_cnot</zorder>
   <zorder>btn_swap</zorder>
   <zorder>btn_cz</zorder>
   <zorder>btn_cphase</zorder>
   <zorder>btn_toffoli</zorder>
   <zorder>btn_cswap</zorder>
   <zorder>label_3</zorder>
   <zorder>btn_qubit</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QGroupBox" name="compileElements">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>951</width>
     <height>101</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(240, 240, 240);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn_createFile">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Create File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calculation">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start Calculation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Filename:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>.txt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filename">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_dropBar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>951</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mode">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Drag and Drop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Current Mode:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dragANDdrop">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Drag and Drop - Mode</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
